Jenkinsfile

```diff
@@ -10,6 +10,12 @@ pipeline {
         buildDiscarder logRotator(artifactDaysToKeepStr: '', artifactNumToKeepStr: '', daysToKeepStr: '', numToKeepStr: '10')
         timeout(time: 90, unit: 'MINUTES')
     }
+    environment {
+        OPENSHIFT_INTERNAL_REGISTRY = "image-registry.openshift-image-registry.svc:5000"
```

Comment:
Should we be exposing this address?

Reply:
this is the default internal openshift address so I think we are safe ;)